packages/teko/examples/HeuristicBlockPermutation/driver-heuristic-permutation.cpp

```diff
@@ -0,0 +1,336 @@
+// @HEADER
+// *****************************************************************************
+//      Teko: A package for block and physics based preconditioning
+//
+// Copyright 2010 NTESS and the Teko contributors.
+// SPDX-License-Identifier: BSD-3-Clause
+// *****************************************************************************
+// @HEADER
+
+// Teuchos includes /*@ \label{lned:being-includes} @*/
```

Comment:
What does this do?

Reply:
Copy pasted from https://github.com/trilinos/Trilinos/blob/15ce71d48654d2c5722d26a8ec93e13afcc88785/packages/muelu/example/basic/Teko.cpp#L10 -- I'll remove.